### DISCUSSION TITLE
How do I do a regex search using meow-visit?

### DISCUSSION THREAD

**Discussion by ksmayur** [meow-edit/meow#477] [Q&A] [answered]
I read here that I could set `meow-visit-sanitize-completion` to `nil`, and do a regex search (instead of looking up symbols or words), but this option does not seem to follow standard regex pattern matching. For example I the case insensitive search for `pattern` using `(?i)pattern` does not show up anything in the mini-buffer. See snapshot below where I get nothing for `(?i)signal` but get all matches using `signal`
<img width="245" alt="image" src="https://github.com/meow-edit/meow/assets/1725447/eb83d65f-c525-4182-ad4c-d47597474fb0">
<img width="194" alt="image" src="https://github.com/meow-edit/meow/assets/1725447/a50da60e-eaa5-446e-9f19-9bed01052872">
@DogLooksGood

**Comment by DogLooksGood** [comment, marked as answer]
Hey,
Is this regexp valid in Emacs? There's a command `toggle-case-fold-search` for such use case.

**Reply by ksmayur** [reply]
Oh wow, I'm new to emacs and I didn't know there's a different format for regexp for emacs until I quickly searched for it after looking at your comment above! Thanks, will try to look for python type regexp matching packages.
